programs/locked-voter/src/instructions/increase_locked_amount.rs

```diff
@@ -74,6 +74,8 @@ impl<'info> Validate<'info> for IncreaseLockedAmount<'info> {
         assert_keys_eq!(self.source_tokens.mint, self.locker.token_mint);
         assert_keys_neq!(self.escrow_tokens, self.source_tokens);
 
+        invariant!(self.escrow.is_not_in_dispute(), "Escrow is disputed");
+
         let duration = unwrap_opt!(
             self.escrow.get_remaining_duration_until_expiration(
```

Comment:
Will we allow disputed accounts to accept the merkle tree airdrops through `increase_locked_amount`? Or do we have a separate mechanism?

Reply:
I think that is separate concern. because our rewards are off-chain, and we can collect new_owner if we want